src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -57,38 +58,53 @@ public Command parseCommand(String userInput) throws ParseException {
         switch (commandWord) {
 
         case AddCommand.COMMAND_WORD:
+            logger.fine(String.format("Add command identified, parsing args..."));
             return new AddCommandParser().parse(arguments);
 
         case EditCommand.COMMAND_WORD:
+            logger.fine(String.format("Edit command identified, parsing args..."));
             return new EditCommandParser().parse(arguments);
 
         case DeleteCommand.COMMAND_WORD:
+            logger.fine(String.format("Delete command identified, parsing args..."));
             return new DeleteCommandParser().parse(arguments);
 
         case ClearCommand.COMMAND_WORD:
+            logger.fine(String.format("Clear command identified."));
             return new ClearCommand();
 
         case FindCommand.COMMAND_WORD:
+            logger.fine(String.format("Find command identified, parsing args..."));
             return new FindCommandParser().parse(arguments);
 
         case ListCommand.COMMAND_WORD:
+            logger.fine(String.format("List command identified."));
             return new ListCommand();
 
         case ExitCommand.COMMAND_WORD:
+            logger.fine(String.format("Exit command identified."));
             return new ExitCommand();
 
         case HelpCommand.COMMAND_WORD:
+            logger.fine(String.format("Help command identified."));
             return new HelpCommand();
 
         case AddNotesCommand.COMMAND_WORD:
+            logger.fine(String.format("Add Notes command identified, parsing args..."));
             return new AddNotesCommandParser().parse(arguments);
 
         case ViewCommand.COMMAND_WORD:
+            logger.fine(String.format("View command identified, parsing args..."));
             return new ViewCommandParser().parse(arguments);
 
         case AddAppointmentCommand.COMMAND_WORD:
+            logger.fine(String.format("Add Appointment command identified, parsing args..."));
             return new AddAppointmentCommandParser().parse(arguments);
 
+        case DeleteAppointmentCommand.COMMAND_WORD:
+            logger.fine(String.format("Delete Appointment command identified, parsing args..."));
```

Comment:
Good addition of loggings which will help with future development processes and debugging processes.